content/getstarted/02-item.md

```diff
@@ -1,56 +1,234 @@
 
-The easiest way to make the changes is to copy this kubevirt directory into a directory of the kubevirt repository, and run the `csi_up.sh` script from there after the cluster is up and running.
+The [Ember-CSI-KubeVirt](https://github.com/embercsi/ember-csi-kubevirt) repository provides a seamless wrapper integrating the deployments of OpenShift or Kubernetes, KubeVirt, [Containerized Data Importer (CDI)](https://github.com/kubevirt/containerized-data-importer), Ember-CSI and a Ceph demo cluster. 
 
-For now the example only supports single node deployments, and for convenience we are using a pod instead of a StatefulSet and DaemonSets and the permissions are too broad.
+Ember-CSI should always be deployed using the [Ember-CSI-Operator](https://github.com/embercsi/ember-csi-operator), like we do here. The operator ensures that all of Ember-CSI's Kubernetes objects such as StatefulSets, DaemonSets, StorageClasses, RBAC rules, etc. are properly configured and running. The Ceph-demo cluster deploys an empty directory backed Ceph cluster within a namespace called `ceph-demo`. We leverage KubeVirt's repository scripts that provide an easy way to deploy OpenShift or Kubernetes using the `make cluster-up` scripts. In addition, CDI is deployed, which provides a declarative way to build Virtual Machine Disks on PVCs for Kubevirt VMs. Finally the Ember-CSI-KubeVirt repository ties all these deployments together into an easy `make all` command. 
 
-Given the following assumptions:
+### Requirements
 
-- This `kubevirt` directory is in `$cl_example`.
-- We have modified `X_CSI_BACKEND_CONFIG` environmental variable in the `csi.yml` (example is for XtremIO).
-- Our current working directory is `kubevirt` repository's root directory.
-- We haven't created a kubevirt cluster yet.
+This demo requires QEMU-KVM, libvirt and Docker installed in the system. In the case where this demo is being deployed in a VM, nested virtualization support must be enabled before proceeding. Nested virtualization for Red Hat/CentOS/Fedora can be enabled using one of the guides [here](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/virtualization_deployment_and_administration_guide/nested_virt) or [here](https://www.linuxtechi.com/enable-nested-virtualization-kvm-centos-7-rhel-7/).
 
-The following commands will create the cluster, make changes to the cluster to support CSI plugins, deploy the CSI plugin, create a PVC, and an app that uses the PVC:
+In Fedora/Red Hat/CentOS:
 
+```shell
+$ sudo dnf -y install qemu-kvm libvirt docker 
+```
+
+Add `docker` group and add your username into the `docker` group.
+```shell
```

Comment:
We also need to run `sudo chown root:docker /var/run/docker.sock` for CentOS, Fedora, and RHEL after the docker service is running.